catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswQueryFactoryTest.java

```diff
@@ -209,6 +215,20 @@
 
     private static List<MetacardType> metacardTypeList;
 
+    public static MetacardType getCswMetacardType() {
+        return new MetacardTypeImpl(CswConstants.CSW_METACARD_TYPE_NAME,
+                Arrays.asList(new ContactAttributes(),
+                        new LocationAttributes(),
+                        new MediaAttributes(),
+                        new TopicAttributes(),
+                        new AssociationsAttributes(),
+                        new MetacardTypeImpl("TestDate",
```

Comment:
this got changed during formatting but it looks like we can just use CoreAttributes.